quinn-proto/src/packet.rs

```diff
@@ -620,13 +640,19 @@ impl PlainHeader {
     }
 }
 
+/// Header of an Initial packet, before decryption
 #[derive(Clone, Debug)]
-pub(crate) struct PlainInitialHeader {
-    pub(crate) dst_cid: ConnectionId,
-    pub(crate) src_cid: ConnectionId,
-    pub(crate) token_pos: Range<usize>,
-    pub(crate) len: u64,
-    pub(crate) version: u32,
+pub struct PlainInitialHeader {
```

Comment:
@Ralith before this becomes public I think we should give this name some more thought? As I understand it this is before removing header protection, which makes the `Plain` prefix a little confusing. Internally we now have `PlainInitialHeader` and `InitialHeader` which could also do with better disambiguation...

Reply:
`PlainInitialHeader` makes people think of plain text, which is unencrypted in context of encryption/decryption.
Maybe we should follow RFC9000/RFC9001 to use the term "Protected" ?

Reply:
I like it! Added a commit that renames `PlainHeader` and `PlainInitialHeader` to `ProtectedHeader` and `ProtectedInitialHeader` -- both of these were previously private so should be fine.

Reply:
SGTM.